src/interpreter.js

```diff
@@ -75,6 +76,7 @@ const Interpreter = async (
             data.object || {},
             ["roles", "users", "everyone"],
             data.array || [],
+            data.arrays || [],
```

Comment:
This definitely can be improved.

Reply:
what exactly i would improve here tho
array is for textsplit
and arrays is for array functions